merkletree/pad.go

```diff
@@ -41,29 +45,42 @@ func NewPAD(policies Policies, key crypto.SigningKey, len int64) (*PAD, error) {
 }
 
 // if policies is nil, the previous policies will be used
-func (pad *PAD) generateNextSTR(policies Policies, m *MerkleTree, epoch uint64) {
+func (pad *PAD) signTreeRoot(policies Policies, m *MerkleTree, epoch uint64) {
```

Comment:
The `policies` argument here is unnecessary, since it's always going to be `pad.policies`, which the `pad` has access to.